web/src/components/navbar/NavBar.tsx

```diff
@@ -20,6 +21,13 @@ export const NavBar = (props: Props): ReactElement => {
   const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
```

Comment:
[boulder] I was hoping your refactor would cover the removal of `useMediaQuery()`.

Reply:
Yeah, alright. I can do that. 

I was imagining like a css style display none for depending on a css media query. Is that what you were thinking?

Reply:
Yep that sounds good!

Reply:
I used inline for one of the 4 states becasue 
<img width="394" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/0ef7778f-bb4f-46f0-9c2c-6482f546004c">
this was overriding my block, and I don't want to touch that because who knows what depends on that existing.

Reply:
<img width="616" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/f4fbe8cf-cffb-4b00-aabf-7504c984f127">
is fixed now though. Will push up in a second